src/components/Layout/Sidebar.tsx

```diff
@@ -26,10 +26,13 @@ const renderItems = (
     {items.map((item, index) => {
       let isItemActive = false;
       let isItemCurrent = false;
+      let isItemNonProduct = false;
 
       if (isInternalItem(item)) {
         isItemActive = currentPath.startsWith(item.to);
         isItemCurrent = currentPath === item.to;
+        isItemNonProduct = isItemActive && !currentPath.startsWith('/minfraud')
```

Comment:
I think the `isItemActive` check here should be removed.  A non-product item can still be active; the styling of which should be contained in CSS and not in the rendering logic.

Reply:
If I remove the `isItemActive` check here, then all nav items get the `.item__nonProduct` class even when they aren't active. Is this OK or should we keep the `isItemActive` check?

Original inquiry: https://github.com/maxmind/dev-site/pull/318#discussion_r817012461

Reply:
I see.  If you replace `currentPath` with `item.to`, you'll get the desired effect of only applying the `item__nonProduct` class to non product links

Reply:
```
isItemNonProduct = !item.to.startsWith('/minfraud') && !item.to.startsWith('/geoip');
```